openedx/core/djangoapps/content/block_structure/migrations/0003_blockstructuremodel_storage.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import openedx.core.djangoapps.content.block_structure.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('block_structure', '0002_blockstructuremodel'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='blockstructuremodel',
+            name='data',
+            field=openedx.core.djangoapps.content.block_structure.models.CustomizableFileField(),
```

Comment:
**Note to Reviewers:** If a new `CustomizableFileField` class wasn't used, the migration would have hard-coded:
```
field=(storages.backends.s3boto.S3BotoStorage(bucket='neem-test'))
```
Or, whatever the configuration said at the time of generating migrations - even though the configuration settings don't affect database queries.  This is a known issue: http://stackoverflow.com/a/41138098